core/src/main/java/org/javarosa/xform/parse/XFormParserFactory.java

```diff
@@ -27,32 +27,11 @@ public XFormParserFactory(Interner<String> stringCache) {
 
     public XFormParser getXFormParser(Reader reader) {
         XFormParser parser = new XFormParser(reader);
-        init(parser);
-        return parser;
-    }
 
-    private void init(XFormParser parser) {
```

Comment:
I did a grep, there are only two uses and they are in javarosa/core/src/main/java/org/javarosa/xform/util/XFormUtils.java